joomla/joomla-cms
# Fixed capitalisation for vote language strings.

Pull Request for Issue #22478 .

### Summary of Changes
Changed the capitalisation for the messages that appear when rating an article.


### Testing Instructions
Only language strings have been change, so a review should suffice.
But if you want to test it:
* Enable the plugin "Content - Vote".
* At "Articles - Options", enable "Show Voting".
* Navigate to an article in the frontend and rate it to see the success message.
* Try to rate the same article again to see the error message.
